easybuild/easyconfigs/l/libnsl/libnsl-1.3.0-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libnsl'
+version = '1.3.0'
+
+homepage = 'https://www.linuxfromscratch.org/blfs/view/svn/basicnet/libnsl.html'
```

Comment:
hm, (b)lfs just provides links and instructions, I think it would be better to use `https://github.com/thkukuk/libnsl` as homepage (sorry if I'm being pedantic)

Reply:
Changed that.